packages/grafana-schema/src/common/logs.cue

```diff
@@ -0,0 +1,4 @@
+package common
+
+LogsSortOrder:     "Descending" | "Ascending"                 @cuetsy(kind="enum")
+LogsDedupStrategy: "none" | "exact" | "numbers" | "signature" @cuetsy(kind="enum",memberNames="none|exact|numbers|signature")
```

Comment:
I am curious, why do we set `memberNames` here if they have the same names as enums, but we don't do it for `LogsSortOrder`?

Reply:
The member names are automatically capitalised in the output TS code, so if we want to preserve their casing we need to define those explicitly. Not necessary for `LogsSortOrder` as those are already capitalised and we want to have them like that in the output.

Reply:
Thank you for explanation. Make sense now. 🙂